### ISSUE TITLE
Build error: idna.h: No such file or directory

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tinselcity/skipfish#3]
```
What steps will reproduce the problem?
1. Download skipfish-1.01b.tgz
2. Extract
3. cd skipfish
4. make

What is the expected output? What do you see instead?

➜  skipfish ✗ make
cc skipfish.c -o skipfish -Wall -funsigned-char -g -ggdb -U_FORTIFY_SOURCE -O3 
-Wno-
format http_client.c database.c crawler.c analysis.c report.c -lcrypto -lssl 
-lidn -lz
http_client.c:38:18: error: idna.h: No such file or directory
http_client.c: In function ‘parse_url’:
http_client.c:275: warning: implicit declaration of function 
‘idna_to_ascii_8z’
http_client.c:275: error: ‘IDNA_SUCCESS’ undeclared (first use in this 
function)
http_client.c:275: error: (Each undeclared identifier is reported only once
http_client.c:275: error: for each function it appears in.)
report.c: In function ‘copy_static_code’:
report.c:744: warning: passing argument 3 of ‘scandir’ from incompatible 
pointer type
make: *** [skipfish] Error 1
➜  skipfish ✗ ls
COPYING       analysis.c    crawler.c     debug.h       report.c      
string-inl.h
Makefile      analysis.h    crawler.h     dictionaries  report.h      types.h
README        assets        database.c    http_client.c same_test.c
alloc-inl.h   config.h      database.h    http_client.h skipfish.c

What version of the product are you using? On what operating system?
1.01b
OSX 10.6.2

➜  skipfish ✗ cc --version
i686-apple-darwin10-gcc-4.2.1 (GCC) 4.2.1 (Apple Inc. build 5646) (dot 1)
Copyright (C) 2007 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

```

Original issue reported on code.google.com by `ric...@gmail.com` on 20 Mar 2010 at 12:06
- Merged into: #2

**Comment by GoogleCodeExporter** [comment]
```
Ack, I missed the other, completely identical issue (Issue 2).  Please close.
```

Original comment by `ric...@gmail.com` on 20 Mar 2010 at 12:08

**Comment by GoogleCodeExporter** [comment]
```
The libidn requirement is only mentioned in the fourth paragraph of the third 
section of the README (which is 
otherwise concerned with how to operate the tool, not how to build it).  
skipfish is very easy to build, but I think 
an INSTALL file would be useful for many.
```

Original comment by `ric...@gmail.com` on 20 Mar 2010 at 12:17

**Comment by GoogleCodeExporter** [comment]
```
It's sort of intentional; I want people to read the documentation, as there are 
several other important points raised there - e.g., dictionary configuration, 
optimal 
settings, etc. Plus, really, this is no different from any other source 
package, where 
you usually have to install dependencies working back from the compile-time 
errors.
```

Original comment by `lcam...@gmail.com` on 20 Mar 2010 at 12:40
- Changed state: **Duplicate**

**Comment by GoogleCodeExporter** [comment]
```
going through the same process - you need to install libssl-dev and 
libidn11-dev (in
addition to libidn11, which you probably already worked out..) - shoudl build 
fine
after that..
```

Original comment by `dsmudger` on 24 Mar 2010 at 4:31
